Reference/Notifications/EditorModel-Notifications/index.md

```diff
@@ -196,6 +196,20 @@ namespace Umbraco.Docs.Samples.Web.Notifications
     NOTE: Dashboards is a collection of IDashboardSlim, each object gives you access to Label, Alias, Properties, whether it's expanded, and whether it IsActive.
     </td>
   </tr>
+  
+  <tr>
+    <td>SendingAllowedChildrenNotification</td>
+    <td>
+      <ul>
+        <li>IEnumerable&ltContentTypeBasic&gt Children</li>
+        <li>IUmbracoContext UmbracoContext</li>
+      </ul>
+    </td>
+    <td>
+    Published right before the allowed children of a node are sent back. (whenever you click create on a node)<br>
+    NOTE: Children is a collection of ContentTypeBasic, each object gives you access to Alias, Description, Thumbnail and more. You can remove or add new children to the list in the notification.
```

Comment:
Instead of the sentence in the parameter, would it make sense to combine the first two sentences?
I'm not entirely sure I understand "whenever you click create on a node" - what is it that happens whenever you click create on a node? 🤔 

```suggestion
    Published right before the allowed children of a node are sent back. (whenever you click create on a node)<br>
    NOTE: Children is a collection of ContentTypeBasic. Each object gives you access to Alias, Description, Thumbnail, and more. You can remove or add new children to the list in the notification.
```

Reply:
I thought the sentence `Published right before the allowed children of a node are sent back` doesn't really make it clear when this is actually called. Whenever you want to create a new node, it'll list you the available node types that you can pick from according to the node that you clicked create on. That is when this notification is sent out. So I was hoping to add some more context to it like that.

Maybe something along the lines of: `Published right before the allowed childrens of a node are sent back. These are the node types that are allowed under a given a node in your content section.`

Reply:
Ah, I see! 🤔 I think I was thrown by the "sent back", not really sure what that meant. But it makes sense now!

I like your second suggestion here, but how about we merge it into just one sentence instead?
Something like:
"Published right before the allowed children of the selected Content Type are sent back during content creation in the Content Section."

Reply:
Good suggestion, I've added that sentence now instead of the other one.